import_logs.py

```diff
@@ -1527,9 +1538,10 @@ def _call(self, path, args, headers=None, url=None, data=None):
             self.RedirectHandlerWithLogging(),
             urllib.request.HTTPSHandler(**https_handler_args))
         response = opener.open(request, timeout = timeout)
+        encoding = response.info().get_content_charset('utf-8')
```

Comment:
This line and the result.decode below caused the issue described here:

https://forum.matomo.org/t/log-file-importer-error-utf-8-codec-cant-decode-byte-0x80-in-position-10-invalid-start-byte/43169